conf.py

```diff
@@ -273,10 +273,11 @@
 # Family Configuration file to use
 family_config_inputfile = f"{FAMILY}/{FAMILY}_{OS}_config.txt"
 
-# Hash table for Replacement Variables
-family_replacevars = {}
-# Hash table for Configuration Values
-family_configvals = {}
+# Hash table for Replacement Variables and Config Values
+family_replacevars, family_configvals = interpretvalues.read_familyvals(family_config_inputfile)
+
+# Unpack replacement variables and place them in the preable for all documents
```

Comment:
`preable` - Just caught this typo, will address with whatever other comments this gets